### PR TITLE
Added User Renewal

### PR DESCRIPTION
**Pull request by IlliaDR**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/UserRenewal/UserRenewal.tsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    };
+
+    const handleSelectChange = (value: any) => {
+        setCityId(value);
+    };
+
+    const sendRenewal = async (data: any) => {
+        await userRenewalsApi.sendUserRenewal(data)
+            .then(() => {
+                if (adminsIds !== null) {
```
Comment (ostappppppp) [inline src/pages/UserRenewal/UserRenewal.tsx:96]: If I'm not mistaken, in TS , it's should work just with `if (adminsIds)`

File: src/pages/UserRenewal/UserRenewalTable/columns.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import { SortOrder } from 'antd/lib/table/interface';
+import { Tag } from 'antd';
+import moment from 'moment';
+import React from 'react';
+
+
+const fetchYears = () => {
+  const arrayOfYears = [];
+  var startDate = Number(new Date().getFullYear());
+  for (let i = 2018; i <= startDate; i++) {
```
Comment (ostappppppp) [inline src/pages/UserRenewal/UserRenewalTable/columns.tsx:12]: Try to avoid magic numbers

File: src/pages/UserRenewal/UserRenewal.tsx (hunk 1 above)
Comment (IlliaDR) [reply src/pages/UserRenewal/UserRenewal.tsx]: Fixed that.

File: src/pages/UserRenewal/UserRenewalTable/columns.tsx (hunk 2 above)
Comment (IlliaDR) [reply src/pages/UserRenewal/UserRenewalTable/columns.tsx]: Fixed it.

**Review by ostappppppp — APPROVED** [review]

**Review by andrii52 — APPROVED** [review]

**Review by VSBalanchuk — APPROVED** [review]

**Pull request merged**